### ISSUE TITLE
Ubuntu PPA does not offer 32-bit MangoHud support

### ISSUE THREAD

**Issue by ChaosBlades** [benjamimgois/goverlay#148]
We should add a blurb to the readme that installing GOverlay through the Ubuntu PPA does not offer 32-bit game support. This is only mentioned on a couple or reddit posts that I can find.

I opened an issue on the MangoHud github for the same issue.
https://github.com/flightlessmango/MangoHud/issues/606

You will need to download MangoHud from the releases tab and run `./mangohud-setup.sh install` in order to enable 32-bit support. After that it should just work.

**Comment by benjamimgois** [comment]
Thanks for the tip @ChaosBlades . I'll add a note to the readme and try to talk to Martin Wimpress to update the ppa.

**Comment by ChaosBlades** [comment]
**Copy Pasta updates from the MangoHud issue.**

> Mangohud was added to the [MPR](https://mpr.hunterwittenborn.com/packages/mangohud/) for Debian and Ubuntu and is build with both 64 and 32bit support. Prebuilds can be found in the [PrebuildMPR](https://github.com/PrebuiltMPR/builder) repository for Debian stable and Ubuntu LTS.

I did get a response from Martin...

> No, 32bit x86 is not a supported architecture in Ubuntu for some years now

Which makes me think he thought I asked for Ubuntu 32-Bit support and not MangoHud 32-bit support on 64-bit Ubuntu. I replied back as such, the reasons why we need 32-bit MangoHud support, and have not gotten a response back after 2 days. Which makes me think I have been ghosted.

It has now been 6 days since I last replied to Martin so definitely got ghosted at this point. I can only guess that Martin has the same viewpoint that Canonical did when they tried to end support for all 32-bit application support in Ubuntu until the community told them how insane they are.
